specifyweb/frontend/js_src/lib/components/QueryBuilder/CreateRecordSet.tsx

```diff
@@ -116,29 +127,16 @@ export function CreateRecordSet({
   );
 }
 
-/**
- * Create a record set from selected records.
- * See also `MakeRecordSetButton`
- */
-export function CreateRecordSetFromQuery({
-  getIds,
-  baseTableName,
-  queryResource,
-}: {
-  readonly getIds: () => RA<number>;
-  readonly baseTableName: keyof Tables;
-  readonly queryResource: SpecifyResource<SpQuery> | undefined;
-}): JSX.Element {
-  const recordSetName =
-    queryResource === undefined || queryResource.isNew()
-      ? undefined
-      : queryResource.get('name');
+function LoadingDialog(): JSX.Element {
```

Comment:
(optional) add optional `header` parameter to `<LoadingScreen />` since we needed that in several places
and then look for usages of `loadingBar` in dialog that can be replaced with it (there would be one in the react workbench I think)